doc/main.qbk

```diff
@@ -25,25 +25,32 @@
 
 [variablelist
   [[
-    [link nudb.overview Overview]
+    [link nudb.introduction Introduction]
   ][
-    An overview of features, requirements, and credits, plus
-    rationale and design information.
```

Comment:
For Beast, having a manually generated table of contents is pretty nice because otherwise with auto-toc you get a thousand type declarations inserted into the ToC. But for NuDB, which has very few declarations, its not so bad. For example look at the list under Reference:
http://vinniefalco.github.io/nudb/

Its not going to grow very much, if at all. I think as long as it is fairly contained to roughly the current size, the existing settings for auto generation of ToC are sufficient.

Compare these pages:
http://vinniefalco.github.io/nudb/index.html
http://vinniefalco.github.io/stage/nudb/ramey/

I like the first one better.